src/index.ts

```diff
@@ -20,7 +20,9 @@ export = (app: Application): void => {
       "pull_request.reopened",
       "pull_request.synchronize",
     ],
-    async (context) => {
+    /* eslint-disable @typescript-eslint/no-explicit-any */
+    async (context: Context<any>) => {
```

Comment:
Codacy found an issue: [Type declaration of 'any' loses type-safety. Consider replacing it with a more precise type.](https://app.codacy.com/manual/tianhaoz/approveman/pullRequest?prid=6311563)